datafusion/core/tests/fuzz_cases/aggregation_fuzzer/fuzzer.rs

```diff
@@ -237,45 +252,53 @@ struct AggregationFuzzTestTask {
 }
 
 impl AggregationFuzzTestTask {
-    async fn run(&self) {
+    async fn run(&self) -> Result<()> {
```

Comment:
the key change is to return an actual `DataFusionError` here and then print it when looking at the task output